.changeset/.npmr

```diff
@@ -0,0 +1 @@
+engine-strict=true
```

Comment:
Shouldn't the file be named `.npmrc` and not `.npmr`? And also be at the root of the repository (and not inside `.changeset`?)

Anyway, since we are forcing `pnpm` already, I don't think we need it, here is the `pnpm` documentation:
https://pnpm.io/npmrc#engine-strict

>engine-strict
>Default: false
>Type: Boolean
>If this is enabled, pnpm will not install any package that claims to not be compatible with the current Node version.

>Regardless of this configuration, installation will always fail if a project (not a dependency) specifies an incompatible version in its engines field.

Reply:
oups maybe I went a bit too fast

Reply:
ohh `.npmrc` is in `gitignore` that is probably what confused me, is it ok if I remove it